components/eam/bld/namelist_files/use_cases/1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs-1pctCO2.xml

```diff
@@ -0,0 +1,147 @@
+----- modified based on 1850 version ---- combined with 1850 use_case for V2
+
+<?xml version="1.0"?>
+<namelist_defaults>
+
+<!-- Set default output options for CMIP6 simulations -->
+<cosp_lite>.true.</cosp_lite>
+
+<!-- Atmosphere initial condition -->
+<ncdata dyn="se" hgrid="ne30np4" nlev="72" >atm/cam/inic/homme/NGD_v3atm.ne30pg2.eam.i.0001-01-01-00000.c20230106.nc</ncdata>
+<!--ncdata dyn="se" hgrid="ne0np4_northamericax4v1" nlev="72" >atm/cam/inic/homme/cami_0001-01-01_northamericax4v1_c190709.nc</ncdata -->
+
+<!-- Solar constant from CMIP6 input4MIPS -->
+<solar_data_file>atm/cam/solar/Solar_1850control_input4MIPS_c20181106.nc</solar_data_file>
+<solar_data_ymd>18500101</solar_data_ymd>
+<solar_data_type>FIXED</solar_data_type>
+
+<!-- 2010 GHG values from CMIP6 input4MIPS -->
+<!-- <co2vmr>284.317e-6</co2vmr> The CMIP6 concentration set by CCSM_CO2_PPMV in cime/src/drivers/mct/cime_config/config_component_acme.xml -->
+<ch4vmr>808.249e-9</ch4vmr>
+<n2ovmr>273.0211e-9</n2ovmr>
+<f11vmr>32.1102e-12</f11vmr>
+<f12vmr>0.0</f12vmr>
+
+<!-- 1pctCO2 increase from 1850 base level -->
+<bndtvghg     >atm/cam/ggas/GHG_CMIP6_1pctCO2_c20180216.nc </bndtvghg>
+<scenario_ghg >RAMPED</scenario_ghg>
+
+<!-- Stratospheric aerosols from CMIP6 input4MIPS -->
```

Comment:
code changes are needed here. currently, when MAM5 is on, the VOLC_CMIP6 will be overwrite as false automatically. We may want to change this.

Reply:
@keziming , you are right, with MAM5, the VOLC_CMIP6 in run time will be false. These settings (including the next block you commented on) are inherited from the intermediate compsets used for the recent v3alpha02 simulations. I believe for the default v3 compsets, the VOLC_CMIP6 block can be removed. Did you mean the settings for f11vmr and f12vmr?

Reply:
@wlin7 I need some clarification here. In the '1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs-1pctCO2' compset, do we want to use the CMIP6 prescribed volcanic optical settings (V2 volcanic setting) or not?

Reply:
@keziming , just an update. The prescribed volcanic settings are removed from all the new use_case files. The GHG settings as you noted below are needed, otherwise the results would be NBFB,